src/codeu/chat/common/User.java

```diff
@@ -51,12 +54,25 @@ public User read(InputStream in) throws IOException {
   public final Uuid id;
   public final String name;
   public final Time creation;
+  public HashSet<Uuid> interestedConvos = new HashSet<>();
```

Comment:
I would change these on the left side to be just `Set` to keep a minimal interface defined.

`public Set<Uuid> interestedConvos = new HashSet<Uuid>();`